src/pages/PostDetail/index.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  IonContent,
+  IonHeader,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+  IonBackButton,
+  IonButtons,
+  IonCardSubtitle,
+  IonCardTitle,
+} from '@ionic/react';
+import React from 'react';
+import Interweave from 'interweave';
+import { RouteComponentProps } from 'react-router';
+import { useResource } from 'rest-hooks';
+import PostResource from '../../resources/post';
+
+type NewsDetailProps = RouteComponentProps<{
+  id: string;
+}>;
+
+const PostDetail: React.FC<NewsDetailProps> = ({
+  match: {
+    params: { id },
+  },
+}) => {
+  const post = useResource(PostResource.detailShape(), { id });
+
+  return (
+    <IonPage>
+      <IonHeader>
+        <IonToolbar>
+          <IonButtons slot="start">
+            <IonBackButton defaultHref="/posts" text="Posts" />
+          </IonButtons>
+          <IonTitle>What's News?</IonTitle>
+        </IonToolbar>
+      </IonHeader>
+      <IonContent>
+        <IonCardTitle>{post.title}</IonCardTitle>
+        <IonCardSubtitle>
+          {post.author} {post.created_on}
```

Comment:
Don't need to create this custom field--there's already a `created_by` field in Directus you can access by specifying the fields: `/_/items/posts/ID?fields=*.*`

Reply:
This fix took 2 hrs to figure out, resolved as per last commit